src/admin/navigations/views/NavigationBarDetail.tsx

```diff
@@ -7,17 +7,17 @@ import { GENERATE_SITE_TITLE } from '../../../constants';
 import useTranslation from '../../../shared/hooks/useTranslation';
 import { withAdminCoreConfig } from '../../shared/hoc/with-admin-core-config';
 
-import './MenuDetail.scss';
+import './NavigationBarDetail.scss';
 
-type MenuDetailProps = DefaultSecureRouteProps<{ menu: string }>;
+type NavigationBarDetailProps = DefaultSecureRouteProps<{ menu: string }>;
 
-const MenuDetail: FunctionComponent<MenuDetailProps> = ({ match }) => {
+const NavigationBarDetail: FunctionComponent<NavigationBarDetailProps> = ({ match }) => {
 	const { tText } = useTranslation();
 
 	const menuId = match.params.menu;
```

Comment:
I'd refactor this to `navigationId` for consistency

Reply:
fixed in commit on pr higher on the stack:
https://github.com/viaacode/avo2-client/pull/1683/commits/1084917d5c4b842be4ff0d47fac51a268eafa026